### DISCUSSION TITLE
Share Your New Phoniebox Designs

### DISCUSSION THREAD

**Discussion by MiczFlor** [MiczFlor/RPi-Jukebox-RFID#2115] [Show and tell]
Hi Phonieboxers,
I've been late this year with asking for images for the infamous "Phoniebox Calendar". Please add your designs here. And feel free to talk about how you did it -- to inspire fellow-Phonieboxers. You can find the last [calendars with links on the homepage](https://github.com/MiczFlor/RPi-Jukebox-RFID#readme).

**Comment by DivineDominion** [comment]
Barebones build in an old tobacco box with large arcade button, using future3

![2023-11-11 08-58 2272](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/59080/b75aa767-aa71-4ae0-abb3-8bb8dbb8594c)
![2023-11-11 08-58 2274](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/59080/794a994c-7cba-47fb-ae50-3952a70dff4f)
![2023-11-11 08-58 2273](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/59080/9a5327e5-d834-48c4-9d96-1380338ff60d)
![2023-11-11 08-58 2271](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/59080/5b778683-b869-44be-a8d6-57cb6a5fd92b)
![2023-11-11 08-58 2275](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/59080/64361c4f-28e9-4358-93cf-893440a82c5e)

**Comment by azsigmon** [comment]
For my daughther, as she says "Teenibox"
![IMG_20230723_142158](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/4508364/ef599a39-5898-458e-a0f5-17369c50f680)
![IMG_20230723_142224](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/4508364/5387d211-fadf-4937-a7d2-d3d26e9beadc)
![IMG_20230726_084756](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/4508364/cf585c01-10f1-4dec-a82d-45763f7327c5)

**Reply by pabera** [reply]
Really cool. Reminds me of my set up. I wonder, is the sound out of the onboard sound good enough? I had trouble with a lot of noise back then which required me to go with a separate sound card.

**Reply by azsigmon** [reply]
Thanks. There is some noise when it is not playing anything but when it is playing it does not bother. Also, I did not manage to get two of my USB soundcards to work 🙈 My problem is more with the power: after one-two days the power bank is empty (even when not running) and needs to be waken up by pressing the button on it, which means opening up the box. Good side is that is runs while on power, of course with the usual restart when plugging in or out. I also had to reinstall everything already twice because of not clear reasons 🤷‍♀️. In the second round the mpd service did not start with some "invalid ELF header" error.

**Reply by AlvinSchiller** [reply]
Which Buttons did you use? I like them :)

> In the second round the mpd service did not start with some "invalid ELF header" error.

Sound like https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/2121 and a broken SD Card.

**Reply by azsigmon** [reply]
The buttons I got from [Amazon](https://www.amazon.de/RUNCCI-spst-drucktastenschalter-momentary-Verriegelung-drucktastenschalter/dp/B07QL1BC23?pd_rd_w=P1A1T&content-id=amzn1.sym.31d3a190-f293-4de7-a7b3-e476444cf936&pf_rd_p=31d3a190-f293-4de7-a7b3-e476444cf936&pf_rd_r=DEZ48JKR4HHZBYK59KJM&pd_rd_wg=5Yedh&pd_rd_r=d2fa4bde-b11e-49cf-b5a3-4ca0639cd421&pd_rd_i=B07QL1BC23&ref_=pd_bap_d_grid_rp_0_5_t&th=1) in a 24 pack.

> Sound like #2121 and a broken SD Card.

My problem came after a while. In both cases the box worked fine but after a few weeks of not using, it did not start correctly anymore.

**Comment by s-raeuchle** [comment]
The xmas gift for my twin daughters.

![phonie_closed_back](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/5b6bd134-1c38-435e-a17d-51e8cf89fc7d)
![phonie_closed_front_II](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/04ed46b5-956b-48cb-9f6a-d6242cf77ed1)
![phonie_closed_top](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/ad6a6509-3b90-4a91-95ae-d097bceccc0e)
![phonie_offen_on](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/7d3a90f1-21bd-4f2c-b4c0-e076399dfdd0)
![phonie_offen_top](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/664773bd-2141-4ac7-9b08-e87ea15bc512)
![phonie_open_front](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/139390972/4b3d5760-722e-427f-aaa8-388d3499853d)

[Here is the configuration](https://github.com/s-raeuchle/Phoniebox)

**Reply by DivineDominion** [reply]
The lid with another lid is a cool idea :)

**Comment by powertomato** [comment]
Just performed a precision delivery for my daughter's Christmas present. The case is 3D printed and covered with felt. The nose and cheeks are play, prev- and next song buttons. A couple of buttons on the back for volume control and maybe some other features in the future.

Quite happy with the result.

![grafik](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/1530983/f06b294f-b417-4802-935b-4f0c7a32f06d)
![grafik](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/1530983/b69a8187-bea2-4c6d-bcba-88890c00a7ba)
![grafik](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/1530983/d6171ea8-858e-4bd5-965b-abc48b2b69e1)
![grafik](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/1530983/dc91eeae-42ed-46a7-83d6-e06502993540)

For the nerdy ones: Electronics is a Rock64 board, needed a couple of python code tweaks, such like a different GPIO library. I wrote a new backend for the card reader (it was not a HUD device)

**Reply by s-martin** [reply]
Really cool box!
I'm sure your daughter will love this present. 🎁


We're always interested in improving our code or new features, so pull requests are welcome, if you want to spend the time! 😉

**Reply by pabera** [reply]
Which version did you install? 2 or 3? Also, did you have issues with the different hardware, beside GPIO?

**Reply by powertomato** [reply]
@pabera Sorry for this late reply, I didn't have time to look much into this. For now I'm running the 2.4 version. But I just installed version 3 on an other spare Rock64 and I'm documenting every change I needed to do. Beside GPIO I was not able to get an SPI card reader running, due to pin assignments (there is a way but I just figured I'd pass on kernel fiddling, since I already had an USB-reader available https://lorenz.brun.one/spi-on-rock64/), audio does not work out of the box and requires some tweaks.

**Reply by pabera** [reply]
Great that you were able to make it work. Since the Phoniebox installation is based on a git clone of this repo, maybe you can push your changes to a personal fork? Would make the documentation simpler..

**Comment by ichrogge** [comment]
The box for my daughter

![20240116_095928](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/114866917/85dab944-a650-4f9f-87f6-b35819781cdf)
![20240116_095952](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/114866917/7f348072-1fe6-4af2-9e7a-e8a31a817997)
![20240115_120514](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/114866917/d5385757-3e4f-4214-b412-dc786b243462)

**Comment by skyhigh24** [comment]
The Box i  built for my niece:
Made with Bend Wood (KerfSpacing) and Denim.
i sacrificed some Logitech Z150, but the metal of the Speakers was pre-bend and it wasnt easily usable, so i used the original Plastic Front from the Speakers to Screw them into the wooden Front  :D
Was a lot of Fun :)

Feel free to ask if you want to know something about it  ;)

![20240309_195524](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/addb638a-1521-4dfe-8d7d-a203aa82cff4)
![PXL_20240309_191210162](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/e30d84bb-c4ce-4568-bbe0-ab816ce29218)
![PXL_20240309_191224464 NIGHT](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/973eb875-698f-4d33-a087-3622f34cebef)
![20240309_171146](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/9137e65b-12fc-456a-ad7d-75a9492c52a7)
![20240309_171134](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/c5f46797-6582-4935-9908-4b9c2aa83020)

![20240309_171144](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/c23fa187-8135-4437-8ea4-c69e0489ae5b)

![20240309_171139](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/66213d21-2d0e-4c14-a251-6fe05bc46497)

**Reply by Oli-79** [reply]
Hi,
I like your screen design. Are you using Future3?
I try to do the same but I'm struggling around as I'm using v2.8 for Spotify. Are you using Future3? If not so, could you share the code?

**Reply by skyhigh24** [reply]
No its also an older Version (according to the install Date it should be 2.5.0 - 4101984b - master)

i just edited the code of the Webinterface for the Screen to size the Elements to my needs. 
i copied the following files, adding "mini" to the name:

/home/pi/RPi-Jukebox-RFID/htdocs/
- inc.load.Controls.php    -> inc.loadControls.mini.php
- inc.controlPlayer.php   -> inc.controlPlayer.mini.php

and also edited the index-lcd.php
Creating the new Files was Critical to not change the Webinterface for Smartphone or PC at the same time, since both would load from the same Files as the screen

I can upload the new files, but it was about nearly a year ago with an older PhonieBOX Version. 
So i cant guarantee that the Files work with the Version 2.8

Since i havnt had access to the Box for the entire Year and it is not connected to the Internet, it was never Updated. 
Also i never got complains about broken Functions. 
it seems to be Stable on THIS PARTICULAR INSTALL (which does not imply it will work on all installs, since i made many little Tweaks on a lot of Files to match my Hardware)


Be aware that i cant access the Box (only the SD-Image) so i cant give you any support on the Files. 

Files Here: 
https://github.com/skyhigh24/phoniebox-2.5-screenmod/tree/main


OH and i used the Method from @splitti to install the Screen:
https://splittscheid.de/phoniebox-display/#ftoc-installation-des-touch-display-alternativer-kiosk-modus
but with my edited index-lcd.php

**Reply by Oli-79** [reply]
Thanks a lot. I will try it.

**Reply by Oli-79** [reply]
![2025-02-28 15 22 35](https://github.com/user-attachments/assets/4c2fb11d-1430-4e31-9247-fd65dbe82819)


Worked out well! Thanks. Only issue right no is the missing cover when using spotify.

**Reply by skyhigh24** [reply]
Looking Good, im glad it works :) 

Does the Spotify Cover work on the Normal Webpage instead of the Mini?

**Comment by alesrosina** [comment]
I've created a wooden box:

![final](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/5054909/18afff55-8801-49bc-b70d-666387de1cea)

And also internals: 

![3](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/5054909/7751bbce-13c2-4a0f-b359-a47b40070a82)


I've written a blog post about it, if you want more details: https://alesrosina.com/2024/03/13/a-speaker-for-my-kid.html

**Reply by X-dark** [reply]
@alesrosina, Looks great, where did you find the rotary encoder cap?

**Reply by alesrosina** [reply]
I've bought it at reichelt.de, but it seems like this exact model is no longer in stock. But a similar one (or even nicer) is: https://www.reichelt.com/potentiometerknopf-fuer-achse-4-mm-silber-men-520-411-p135553.html

**Reply by X-dark** [reply]
Thanks. Yes I saw that one but was finding yours nicer

**Reply by skyhigh24** [reply]
@X-dark Take a look at this: https://de.rs-online.com/web/p/potentiometer-knopfe/4659842 As far as i know the KY-040  should have a 6mm shaft. Does look like the one @alesrosina used. 

In the Shop they have many more Caps with nice looking Designs, even brushed Aluminium

**Comment by skyhigh24** [comment]
to test the basics before i built the Big PhonieBox i bought everything twice (also to have a spare part if i damaged something). 
the first time i used the cheapest speakers and buttons i could find, but didnt like them in the end.
 
Since i´d had everything twice and was bored today i built a mini Version. 
The Big one isnt really designed to be Portable, so this one is. 
also its way less work to build :D 

![PXL_20240329_164802558](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/edff5382-2091-4426-9519-88a6af2d35b7)

![PXL_20240329_164722777](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/107815792/3d49c45d-4719-4d83-a7b4-b033d0c51b51)

The Buttons are Color Coded the same as the Big Box.
The Volume Rocker of the Mini is on the Inside, the Cheap Speakers only included an "In Cable" Potentiometer.

**Comment by luisete79** [comment]
Dear all, I finally finished mine. My kids seem to love it. I decided to go with something rather simple and easy to use with only one button Comments and questions are more than welcome. Thanks to all who contribute to this fantastic project.
![PXL_20240510_192517652](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/161035081/64100b5a-b536-4d21-ae11-6b3170e052de)
![PXL_20240515_164131344](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/161035081/07243633-ed0d-4ad8-85f0-6ecccbc4cebb)
![PXL_20240515_164117501](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/161035081/39266e57-61c0-4db3-9cfc-55bf6621ea5c)

**Reply by pabera** [reply]
Did you create the wooden box yourself?

**Reply by luisete79** [reply]
No, they are actually 2 wooden trays joined together. I bought them in Action for about 2 € each. Then, of course I had to make the holes for the speaker, sand everything, and paint it, because the front and the back is supposed to be the base of the tray and is not as wooden and nice as the interior.
![Capture](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/161035081/541bf68c-628c-4ecb-bdd9-7090488fe35c)

**Reply by ghost** [reply]
What kind of speakers you was using for?

**Reply by skyhigh24** [reply]
@JoniArida depending on the Picture its a Sony TV Speaker from the KDL-32P2530 you can buy it with its part number: 1-826-479-11

**Comment by anemes** [comment]
![PXL_20240516_114836806](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/260616/ea219812-7e0d-4143-afca-6e21b8ca0cdb)

I looked around for a fun kids toy that I could pull apart for a while, and the one I landed on turned out to be already gutted by F&P themselves: https://x.com/Foone/status/1454230585933631488

Kids really like designing the record covers. This has the RFID under the yellow record platter. It would have been cooler to put it into the tone arm, but also more annoying for the kids.

**Reply by pabera** [reply]
this is a really cool idea!!

**Reply by s-raeuchle** [reply]
Wow thats a beauty! Well done!

**Comment by X2423** [comment]
this is my phoniebox - I decided on this box because it had a lockable section for the components and a carry handle. I made speaker holes through the hard shell but it is masked by the fabric for a clean finish 

![IMG_8033](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/171313596/07996532-10e3-47cb-96b0-e9a9f711d421)
![IMG_8038](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/171313596/5341d869-64cb-4711-81d1-19b2356ecaaa)
![IMG_8039](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/171313596/25f6b8b7-d1a1-4609-a725-2b64146c6e64)

**Comment by d4rkd3v1l** [comment]
We already built our first Phoniebox last year, but didn't really manage to post it here. So now, after we built our 2nd one, it's time to post em both. 😊 

# Voxiebox
A Vox Pathfinder 10 guitar amp used as a case for our first Phoniebox. We just removed the old amp and speaker and put all the Phoniebox stuff in. The covers on the top (card slot and buttons) and backside are 3d printed.
![IMG_4848](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/3475036d-6666-4196-ad56-860365a28fd8)
![IMG_4853](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/95e1de4e-db40-46dd-abd5-88d7212fdcfc)
![IMG_4935](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/f9321427-e409-44b7-97d8-ac51252256e1)

# Owlybox
We found an awesome owl model on thingiverse and decided to make another Phoniebox out of it. 
![IMG_6892](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/9d82adbd-82dd-4960-a5fb-edde4e501d86)
![IMG_6881](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/26de0fb8-ea73-4e42-aaf0-3e31393854c0)
![IMG_6884](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/13336238/d0a0178c-6318-4aad-ab46-922faf902c8e)

For more details, see https://www.thingiverse.com/thing:6678877.

**Comment by Yo55arian** [comment]
Sharing my Phonebox that was built as a kid's Christmas present. Very popular so far! 

Fairly simple build based on a spare Raspberry Pi 3b. Everything is contained in a mini toolbox from our local hardware store. I used a Pimorino on/off shim wired to the rear LED momentary switch. I added a green status LED to the top but placed this so it could be hidden by folding down the handle to ensure it didn't interfere with getting them off to sleep. 

Only thing still needed is a way to stop inquisitive fingers from opening it up! I'm considering a small cable tie through some drilled holes but other suggestions are very welcome.

A big thank you to the Phoniebox community. This is a really great project that has clearly brought a lot of joy to families all over the world.

![IMG_20241226_174238](https://github.com/user-attachments/assets/4febc010-03ca-4fe0-b07e-54e99a00e4d6)
![IMG_20241226_175125](https://github.com/user-attachments/assets/6743b6d5-bf06-4b8f-9e6c-d9be672b900d)
![IMG_20241226_175413](https://github.com/user-attachments/assets/1ff01675-3314-4350-bd69-4c3da6968d51)
![IMG_20241226_175527](https://github.com/user-attachments/assets/813b3705-66e5-4496-a874-b48763b604e8)

**Comment by Bronkoknorb** [comment]
### Hermibox (Police bus)

Finished it some time ago already as a present for my nephew. Finally found the time to share it.

![photo](https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo.jpg)

<img src="https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo2.jpg" width="180"> <img src="https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo3.jpg" width="180"> <img src="https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo4.jpg" width="180"> <img src="https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo5.jpg" width="180"> <img src="https://github.com/Bronkoknorb/hermibox/raw/main/photos/photo6.jpg" width="180">

More details: https://github.com/Bronkoknorb/hermibox/

**Comment by davidjdewitt** [comment]
I am unable to get my PhonieBox V2.8 to play a Spotify playlist.   The player asserts "Spotify is playing" but no sounds emerge.  Since 2.8 supposedly works with Spotify it is clearly my fault but I was hoping someone who has Spotify working could help me out.  MP3s play just fine.   My email is david.dewitt@outlook.com.  I am a computer science professor (University of Wisconsin) so not a total idiot.
Thanks

**Comment by X-dark** [comment]
Hi,

Here is what we built with my wife, for our daughter, as a Christmas present.

![IMG_20241220_110307](https://github.com/user-attachments/assets/c8ab170f-5616-4129-9c02-7653be3cde7c)
![IMG_20241220_110323](https://github.com/user-attachments/assets/56857cf2-814c-4a3f-86b1-f385ff1bdc67)
![IMG_20241220_110332](https://github.com/user-attachments/assets/3ec94f83-a063-4b3f-b3c2-7ca177ad7e06)
![IMG_20241220_110341](https://github.com/user-attachments/assets/52f3d1c2-a6ee-40a6-ad7e-a54a54c808f4)
![IMG_20241220_110351](https://github.com/user-attachments/assets/674f3a09-3b71-416b-8744-46b8a23b0227)
![IMG_20241220_105542](https://github.com/user-attachments/assets/1cf189f1-31cc-4ceb-b850-9d7bf9df6b79)

Main components are:

* Raspberry Pi Zero 2 WH
* On-Off SHIM
* HifiBerry MiniAmp
* AZDelivery RC522
* Visaton FR7
* ESSENTIELB Powerbank 10000 mAh

**Comment by gregoiremercier** [comment]
Hello,

I've played with Phoniebox for the last month to build one for my 3 years old son.

# What a tried and issues
Keep in mind that this happened when i tried it. Things may have been fixed since then or other issues can occured
I tried all the versions of Phoniebox and end up with the master version of v2
The one liner never worked for me (all versions) and i had to download the source manually, copy them, change the scripts permission and then run the installer

## Future 3 - Master
- UI installer was not working, managed to recompile it myself but it was a pain
- RFID USB installer was broken, skipped it during installed, edit the scripts using the fix on develop and then manually run it
- Never managed to make the USB RFID work (Neuftech https://www.amazon.fr/dp/B018OYOR3E)

## Future 3 - Develop
I then tried the develop version as I found out that the RFID USB installer was fixed there
- UI installer was not working, managed to recompile it myself but it was a pain
- Never managed to make the USB RFID work (Neuftech https://www.amazon.fr/dp/B018OYOR3E)

## Current 2 - Develop
Tried to go back to the version 2 to make the USB RFID work.
- Never managed to make the USB RFID work (Neuftech https://www.amazon.fr/dp/B018OYOR3E)
- Had a lot of GPIO errors

## Current 2.8 - Master 
Went back to the stable version due to the GPIO errors
- Never managed to make the USB RFID work (Neuftech https://www.amazon.fr/dp/B018OYOR3E)
- Tried a Whadda WPI405 Kit RFID (MFRC522) successfully

## Solution chosen
Face with the Future 3 installer issues and the v2 develop GPIO issues, i choose to stay on the v2.8 master version.
I also found multiple blog detalling the building of a phoniebox and as they were a few years old, they were all referencing Phoniebox 2, so it was easier to replicate with their help.
I'm using the Spotify version as i find it easier to add new tracks instead of having to get the mp3 from somewhere

# The build

Reuse a Pax patrol truck to buld this box
The On/Off button is connected to the OnOff Shim (button & led) and light up as soon as the power is up even if phoniebox is not running
The 3 others buttons are connected to GPIO and reflect the LedStatus from Phoniebox GPIO
![All buttons](https://github.com/user-attachments/assets/63b7ff65-51f9-488f-8459-f512eeb37d2b)

All the cards are customised to the associated playlist and fit in the trunk
![All cards](https://github.com/user-attachments/assets/90df84d8-b624-4b8e-9120-35936e5b2fa8)
![Cards](https://github.com/user-attachments/assets/15481b35-eeff-45b4-a1e6-0ab1a6a4d007)

The RFID reader is integrated at the top of the truck and can start/stop the music
![Play](https://github.com/user-attachments/assets/685566cc-727f-44c9-9a81-93e6be287ae7)

The battery fit in the front of the trunk. In the end it was to big to fit with the rest, but i find it better as i can see the battery indicator and i didn't have to add an usb extension to charge it from the outside
![Battery](https://github.com/user-attachments/assets/6a0c257e-f34c-4d12-8512-f1c022b2fad8)

# The connections

Everything fit in this.
![Inside](https://github.com/user-attachments/assets/ca10a6d7-e21d-4573-8d07-a6ddfaff7078)

I started by testing the whole systeme through bread board and jump cable.
I then move that to perforated cards and soldering.
As I'm not a solder expert, so through manipulation during testing and building, i add attachement breaking again and again after redoing them. So once it was validated, i put a lot of glue on all of them to securise them

Here the GPIO configuration i ended up with
![image](https://github.com/user-attachments/assets/4d4c9fb9-83be-4339-96e8-57b3fb9a1441)

Here the Button setup with resistor i ended up with
![image](https://github.com/user-attachments/assets/3a307a01-9b0e-41c5-af1c-78d413c51346)

# The components
- An old Paw patrol truck found for 1€ in a garage sale
- Rasberry PI Zero 2 WH
- Hifiberry Miniamp
- 2 x Visatron 7 
- 4 x EG STARTS Arcade buttons with led
- OnOff Shim from Pimoroni
- GPIO Extension Board to ease the connections (from AZDelivery)
- Whadda WPI405 RFID (MFRC522): it was the one sold from my local electronic shop
- RFID 13.56 MHz cards
- Printable stickers to customize the cards (https://www.amazon.fr/dp/B09VLFRK3X). They are almost the size of the cards and they offer a word template on their website to print them easily?
- A small battery that offered passthrough (i want to avoid my son tantrum when there is no more battery when he wants to listen to music) and had enough reviews (https://www.amazon.fr/dp/B08VD632WJ)

# The reference

I used the documentation, manual and every resources available on this git.
I used a lot of information from various website to achieve this but the two main one were:
https://splittscheid.de/phoniebox-bauanleitung-toniebox-alternative/
https://koboldimkopf.wordpress.com/2020/01/10/tutorial-phoniebox/

# Thanks
Thanks a lot for this project. I had a lot of fun building and playing with it.
And now my son wants to sleep with the truck in is bed, so saying he is happy doesn't quite discribe it.

**Comment by saveli** [comment]
![IMG_20250728_191635](https://github.com/user-attachments/assets/00a105e1-7700-404e-b114-03acb44cf76b)
![IMG_20250728_191645](https://github.com/user-attachments/assets/3224fbc2-ebf7-45b8-ba89-e5fadc22d15b)
![IMG_20250728_192056](https://github.com/user-attachments/assets/6f9061da-0736-4e2b-b54a-33dca54fab72)

Key components:
- Version 2 with Spotify Integration
- RPi 3 B+
- PN532 
- Xlayer Powerbank All-in-One 10000
- Speedlink Twoxo
- FeinTech Ground Loop Isolator ATG00101
- Ikea Kuggis with wooden lid

Notes:
- everything is mounted to the wooden lid with holes in the case for 
  - power on/off button
  - USB-C charging the powerbank
  - two screws to lock the box
- RC522 never worked
  - tried a second RC522
  - C minimum working examples also dysfunctional
- the powerbank is based on a discussion recommendation
  - yes, it does have Pass-Through-Charging
  - but the switchover time when plugging in or unplugging resets the RPi
  - adding a capacitor (3900µF, 16V) didn't help
  - the voltage is also a bit too low, so RPi throttles (warnings in the info tab)

**Reply by skyhigh24** [reply]
looking adorable 👍

**Comment by sebascha** [comment]
![89](https://github.com/user-attachments/assets/3f5e06f3-fa2c-4142-b8dd-8df2bcfb1ea2)
![90](https://github.com/user-attachments/assets/4f9cfa89-e6ba-4ea9-a385-3bdc48f88441)
![87](https://github.com/user-attachments/assets/1fc5fd69-79f4-42c2-9fe7-8cf164925913)
![88](https://github.com/user-attachments/assets/63e7ec3a-6ff3-4b96-9c3c-337133d1152f)

More of a basic phoniebox with flower decals.

-USB-C chargeable
-NFC Tag on top with link to phoniebox website for easy control

**Comment by luisete79** [comment]
As @skyhigh24 mentioned, it is and old speaker of a broken TV.

**Comment by XtracT** [comment]
<img width="597" height="597" alt="image" src="https://github.com/user-attachments/assets/677417d4-d06d-492c-a194-042c146e7a4a" />

This is my son's owlbox, I see there is an owl theme going on here! 

If anyone is interested in 3D printing it, everything can be found in makerworld: https://makerworld.com/en/models/1914879-owlbox-rfid-tag-speaker-for-kids-phoniebox#profileId-2053948

**Comment by Striker62** [comment]
I took an old speaker from a TV Blu-ray player and installed a Raspberry Pi 4 inside. I then connected it to a sound HAT and a USB RFID reader. I used an Android tablet as a screen. Additionally, I connected a USB webcam and installed the Photobooth app. The idea is to use it at children's birthday parties or in daycare. Both apps run simultaneously, and the Phoniebox is controlled via RFID. The challenge was that the Photobooth app runs on 64-bit systems. So, I made the Phoniebox software 64-bit compatible and installed everything on my Trixie device. Next, I'll add my Canon Selphy CP1500 printer so the photos can be printed directly.

<img width="480" height="640" alt="grafik" src="https://github.com/user-attachments/assets/03af14c7-bc45-45ec-b851-4853c7312ed8" />
<img width="480" height="640" alt="grafik" src="https://github.com/user-attachments/assets/a15b8a86-166e-428f-80fe-57e061f103d6" />

**Comment by topas-rec** [comment]
### Modular Phoniebox (... to be continued)

I built a second box, this time inside a old 3,5 " hard drive external housing.

![2026-01-12-20-24-14-193](https://github.com/user-attachments/assets/e9a7116f-a577-4b84-a2db-4bcab46558c7)
![2026-01-12-20-24-43-178](https://github.com/user-attachments/assets/74acf1cc-f303-4f19-b591-f12f21c7796c)

Most parts of the external hard drive cage were reused, like the power button, the USB port (now an outlet) and the power LED (currently off).

It can be used with headphones only currently, but has **a USB power output** and of course a headphone output.

I'll build an additional amp box with just speakers and an amp with USB power input and headphone jack input.
This smaller box can then be attached and connected (USB and audio jack) to the amp box and then forms a fully functional phoniebox.
This way the phoniebox can be split in small and large part and can be used in a smaller size on the go (on while traveling in the car) while keeping a proper box for listening without headphones at home.

Getting everything in the drive housing was a challenge.

![2026-01-18-19-31-51-784](https://github.com/user-attachments/assets/514dd9e3-00e2-4f59-8936-78f661d15bcf)

Final (half of the) box:

![2026-01-22-17-33-43-360](https://github.com/user-attachments/assets/4065154a-d2b4-4875-9df6-3e7082301177)

**Reply by topas-rec** [reply]
Here is the final (though unpainted) box including a dump amp box carrying the previously shown box holding the digital part of the box.

![2026-03-10-21-34-08-277](https://github.com/user-attachments/assets/669af7e4-89a6-4636-a64a-540182de82dd)
![2026-03-10-21-34-16-706](https://github.com/user-attachments/assets/56ee5701-b182-43ed-bf36-e4b55c96e1b4)

**Comment by pat1977j** [comment]
Challenging but successful.
I used a defective Toniebox to build my own Phoniebox. The original power supply could be reused (9V → 5V), and a rechargeable battery was also installed (2500 mAh).
In addition, an MPU6050 gyro/accelerometer was integrated. This allows skipping to the next track by lifting to the left, and going back to the previous track by lifting to the right.
<img width="4032" height="3024" alt="IMG_2128" src="https://github.com/user-attachments/assets/2cf75fa3-2f40-4c34-92fe-a07c7a834eda" />
<img width="3024" height="4032" alt="IMG_2129" src="https://github.com/user-attachments/assets/78792436-167c-4045-be74-b7fbd7330e72" />
<img width="3024" height="4032" alt="IMG_2130" src="https://github.com/user-attachments/assets/f733cc86-8c14-4b11-b82f-ab45c8e2fe80" />
